contracts/SnappAuction.sol

```diff
@@ -265,4 +268,46 @@ contract SnappAuction is SnappBase {
             appliedAccountStateIndex: 0
         });
     }
+
+    function orderBatchUpdate() private {
+        if (
+            auctionIndex == MAX_UINT ||
+            auctions[auctionIndex].size == maxUnreservedOrderCount() ||
+            block.timestamp > (auctions[auctionIndex].creationTimestamp + 3 minutes)
+        ) {
+            createNewPendingBatch();
+        }
+    }
+
+    function unpackAndVerifyOrderData(bytes memory packedOrder) private view {
+        // TODO - use ECRecover for signatures from first 16 bits of packedOrder
```

Comment:
When using ECRecover we need to attach a signature to each order, which is 65 bytes of extra data. This will allow us to recover the ethereum public address which we can then use to look up the accountId. Check https://github.com/protofire/zeppelin-solidity/blob/master/contracts/ECRecovery.sol for more info on how ecrecover works.

Reply:
Thanks, although I was aware of how ECRecover works. I suppose the 16 bits is what threw you off.

Reply:
Yes, the comment is weird. It sounded as if you want to recover something from 16 bits, where really you need 65 bytes of data and can only recover the main Ethereum address which still needs to be looked up. Can you adjust the comment?

Reply:
```suggestion
        // TODO - use ECRecover from signature contained in first 65 bytes of packedOrder
```